activiti-steps/activiti-steps-assertions-api/src/main/java/org/activiti/steps/matchers/SequenceFlowMatchers.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.steps.matchers;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.activiti.api.process.model.events.BPMNSequenceFlowTakenEvent;
+import org.activiti.api.process.model.events.SequenceFlowEvent;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class SequenceFlowMatchers {
+
+    private String definitionKey;
+
+    private SequenceFlowMatchers(String definitionKey) {
+
+        this.definitionKey = definitionKey;
+    }
+
+    public static SequenceFlowMatchers sequenceFlow(String definitionKey) {
+        return new SequenceFlowMatchers(definitionKey);
+    }
+
+    public OperationScopeMatcher hasBeenTaken() {
```

Comment:
@erdemedeiros I was thinking that we can check on sequence flows based on the two bpmn activities that they are connecting.. we have all the information to check that.

Reply:
@salaboy I'm not sure I got your point here. Do you mean use source and target to describe the sequence flow (instead of its key)?

```
.expect(processInstance().hasBeenStarted(),
                        startEvent("StartEvent_1").hasBeenStarted(),
                        startEvent("StartEvent_1").hasBeenCompleted(),
                        sequenceFlow("SequenceFlow_1uccvwa").hasBeenTaken(),
                        task("Task User1").hasBeenCreated(),
                        task("Task User1").hasBeenAssigned())
```
-->

```
.expect(processInstance().hasBeenStarted(),
                        startEvent("StartEvent_1").hasBeenStarted(),
                        startEvent("StartEvent_1").hasBeenCompleted(),
                        sequenceFlow("StartEvent_1, "Task User1").hasBeenTaken(),
                        task("Task User1").hasBeenCreated(),
                        task("Task User1").hasBeenAssigned()